core/src/main/resources/error/error-classes.json

```diff
@@ -1599,6 +1599,11 @@
     },
     "sqlState" : "42703"
   },
+  "UNRESOLVED_CUSTOM_CLASS": {
```

Comment:
Is it better to put the class name that cannot be parsed in the message?

Reply:
I think there two different case, 1. class name is null 2. class name isn't null, but can't be resolved. The `UNRESOLVED_CUSTOM_CLASS` only suite for case 2. Case 1 should be handle before resolve class name. @MaxGekk WDYT?

Reply:
In my opinion, it seems impossible to cause the error to occur unless the class is null. Also when I try I non existing class or a class not defined in the corresponding PATH I get: ClassNotFoundException. What do you guys think about letting the error without the class and maybe adjusting the message?